docker-compose/goloop2moonbeam/gochain/Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM goloop/gochain-icon:0.9.7
+ARG GOCHAIN_IMAGE=goloop/gochain-icon:latest
+FROM ${GOCHAIN_IMAGE} as gochain
 
+RUN apk add --no-cache jq curl
 
 ENV PROVISION_CONFIG=/gochain/provision/config
 ADD config $PROVISION_CONFIG
```

Comment:
`RUN echo "0x3" > $PROVISION_CONFIG/nid.icon`

Shouldn't hardcode this, instead get this from genesis.json